content/asp.net/getting-started/files.md

```diff
@@ -3,13 +3,76 @@ by [Steve Smith](http://deviq.com/me/steve-smith)
+
+```c#
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+
+namespace ConsoleApplication
+{
+    public class Startup
+    {
+        public void Configure(IApplicationBuilder app)
+        {
+            app.Run(context =>
+            {
+                return context.Response.WriteAsync("Hello from ASP.NET Core!");
+            });
+        }
+    }
+}
+```
+
+The ``app.Run`` method configures middleware, which you'll learn more about in the next lesson. In addition to configuring your app's request pipeline, the ``Configure`` method is also used to (surprise!) configure other aspects of the app. It's here that you'll add support for diagnostics, logging, error handling, routing, and frameworks like MVC. These packages often have helper methods that follow a naming pattern of ``Use[Package]``, which is used to configure them within the ``Configure`` method. For example, configuring MVC is done with the ``app.UseMvc();`` statement. You'll see examples of these packages in later lessons in this tutorial. 
+
+### ConfigureServices
+
+The ``ConfigureServices`` method in the ``Startup`` class is optional. You use it to populate the services *container* used by ASP.NET Core. This container provides requested services to objects within your app through a process called [dependency injection](http://deviq.com/dependency-injection/). The container is represented by the ``IServiceCollection`` parameter that is passed into the ``ConfigureServices`` method. ASP.NET Core provides and requests many services itself, and uses the same infrastructure you will use in your app. Many built-in as well as third-party features require certain services to work properly, and you'll often see a particular package (for instance, MVC) adding services in ``ConfigureServices`` and also being configured in ``Configure``.
+
+Some packages will require many different services. Rather than requiring many statements to register each service individually, helper methods are added. By convention, these helper methods follow a naming pattern of ``Add[Package]``. So, for example, to add the services needed by MVC, you would add the statement ``services.AddMvc();`` to ``ConfigureServices``.
+
+## Other files
+
+Your ASP.NET Core app also requires a project file. However, most of the time you won't need to work with this file directly, and in any case its structure will be changing before .NET Core's tooling (which includes the command line interface as well as editors like VS Code and Visual Studio) are finalized. For now, you'll notice a *project.json* and probably a *project.json.lock* file in your application folder.
```

Comment:
Makes sentence really long, and thus harder to parse. Breaking it up with full stops makes it more accessible, especially to non-native English speakers/readers.
See also: http://english.stackexchange.com/questions/79/is-it-acceptable-to-start-a-sentence-with-however
@benrick